pkg/cmd/server/kubernetes/node/node.go

```diff
@@ -140,7 +140,7 @@ func (c *NodeConfig) HandleDockerError(message string) {
 		glog.Fatalf("error: %s", message)
 	}
 	glog.Errorf("WARNING: %s", message)
-	c.DockerClient = &dockertools.FakeDockerClient{VersionInfo: dockertypes.Version{APIVersion: "1.18"}}
+	c.DockerClient = &dockertools.FakeDockerClient{VersionInfo: dockertypes.Version{APIVersion: "1.22"}}
```

Comment:
If we have constant `minimumDockerAPIVersionWithPullByID `, why we don't use it here?